tracing_tools/perf/cpu.sh

```diff
@@ -2,7 +2,12 @@
 
 # Usage: sudo cpu.sh pid
 
-perf record -F 99 -p $1 -g -- sleep 60
-perf script > out.perf
-/opt/FlameGraph/stackcollapse-perf.pl out.perf > out.folded
-/opt/FlameGraph/flamegraph.pl out.folded > cpu.svg
+if [ "$#" -ne 1 ]; then
+  PID=`pgrep tikv-server | head -n 1`
```

Comment:
this script is not only for TiKV

maybe you can add another tikv_cpu.sh instead

Reply:
Good idea. Updated.